### DISCUSSION TITLE
IP Calculator Addon

### DISCUSSION THREAD

**Discussion by PieterL75** [netbox-community/netbox#7689] [General]
I missed an IP Calculator that showed what subnetmask a CIDR is again..
My brain fails to memorize them :-)
there are a lot of them on the internet, but I just did not like opening a new page each time..

I had this Javascript based thingy laying around for a long time. It is a bundle of little scripts that I found on the internet.
It was glued together with a Netbox Plugins and published to PyPi

https://pypi.org/project/netbox-ipcalculator/

It supports IPv4 and IPv6.

Have fun with it !

Pieter

**Comment by jeremystretch** [comment]
Nice, I'll add this to the list [on the wiki](https://github.com/netbox-community/netbox/wiki/Community-Contributions)!
